OpenKeychain/src/main/java/org/sufficientlysecure/keychain/network/OkHttpClientFactory.java

```diff
@@ -65,8 +65,11 @@ public static OkHttpClient getClientPinnedIfAvailable(URL url, Proxy proxy) {
 
         // If a pinned cert is available, use it!
         // NOTE: this fails gracefully back to "no pinning" if no cert is available.
-        if (url != null && TlsCertificatePinning.getPinnedSslSocketFactory(url) != null) {
-            builder.sslSocketFactory(TlsCertificatePinning.getPinnedSslSocketFactory(url));
+        TlsCertificatePinning tlsCertificatePinning = new TlsCertificatePinning(url);
```

Comment:
this had a null check before, is that no longer relevant?

Reply:
``TlsCertificatePinning.getPinnedSslSocketFactory(url) != null`` was used to check if a Pin is available, this has been replaced with ``tlsCertificatePinning.isPinAvailable()``.